docs/home/contribute/style-guide.md

```diff
@@ -144,14 +142,43 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+### Versioning Kubernetes examples
+
+Example that include version specific information should be matched with accompanying text and the relevant Kubernetes version identified.
```

Comment:
Suggestion to make the language a little more direct:

Code examples and configuration examples that include version information should be consistent with the accompanying text. Identify the Kubernetes version in the **Before you begin** section.

To specify the Kubernetes version for a task or tutorial page:

* Include `min-kubernetes-server-version` in the front matter of the page.
* In the **Before you begin** section, use `{% include tasks-tutorial-prereqs.md %}.

Reply:
thanks - updating